migrations/versions/811f4226a52a_.py

```diff
@@ -1,32 +1,23 @@
 """empty message
```

Comment:
You might need to git fetch and git pull in your main branch. There shouldn't need to be any changes to this file